src/Components/EntryPopup.jsx

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react'
+import axios from 'axios'
+import Geocode from 'react-geocode'
+import moment from 'moment'
+import { Container, Image, Icon, Header } from 'semantic-ui-react'
+
+class EntryPopup extends Component {
+
+  state = {
+    caption: '',
+    category: '',
+    created_at: '',
+    image: '',
+    latitude: '',
+    longitude: '',
+    address: ''
+  }
+
+  async componentDidMount() {
+    await axios.get('/api/v1/posts/' + `${this.props.id}`).then(response => {
+      this.setState({
+        caption: response.data.caption,
+        category: response.data.category,
+        created_at: response.data.created_at,
+        image: response.data.image,
+        latitude: response.data.latitude,
+        longitude: response.data.longitude
+      })
+    })
+    this.geolocationDataAddress()
+  }
+
+  geolocationDataAddress = () => {
+    Geocode.setApiKey(process.env.REACT_APP_API_KEY_GOOGLE_MAPS)
+    Geocode.fromLatLng(parseFloat(this.state.latitude), parseFloat(this.state.longitude)).then(
+      response => {
+        const address = response.results[0].formatted_address
+        this.setState({ address: address })
+      },
+      error => {
+        console.error(error);
+      }
+    )
+  }
+
+  executeOnClick(isExpanded) {
+    console.log(isExpanded);
+  }
```

Comment:
do we need this now that `react-show-more` is removed?

Reply:
nope. take it away.